setup.py

```diff
@@ -0,0 +1,26 @@
+from setuptools import setup
+
+setup(name = 'safetyculture-sdk-python',
+      version = '2.4.2',
+      description = 'iAuditor Python SDK and export tools',
+      url = 'https://github.com/SafetyCulture/safetyculture-sdk-python',
+      author = 'SafetyCulture',
+      author_email = 'integrations@safetyculture.io',
+      include_package_data=True,
+      packages = ['safetypy', 'tools', 'tools/exporter'],
+      entry_points = {
+            'console_scripts': [
+                  'iauditor_exporter = tools.exporter.exporter:main',
```

Comment:
If you rebrand here you need to rebrand everywhere

Reply:
went ahead and rebranded through README (other than the name of the overall SDK which remains SafetyCulture SDK Python)